openpype/hosts/tvpaint/api/communication_server.py

```diff
@@ -834,8 +835,9 @@ def execute_george_through_file(self, george_script):
 
 
 class QtCommunicator(BaseCommunicator):
+    title = "AYON Tools" if AYON_SERVER_ENABLED else "OpenPype Tools"
     menu_definitions = {
-        "title": "OpenPype Tools",
+        "title": title,
```

Comment:
Should this actually be using the `AVALON_LABEL` key so it can overridden by a studio?

Should we also update this default value here:
- https://github.com/ynput/OpenPype/blob/9369d4d931f8bd9f7e02e480099d64a4bb89f9b9/openpype/hosts/tvpaint/tvpaint_plugin/plugin_code/library.cpp#L490

Reply:
Also:

- https://github.com/pypeclub/OpenPype/blob/9369d4d931f8bd9f7e02e480099d64a4bb89f9b9/openpype/hosts/tvpaint/tvpaint_plugin/plugin_code/library.cpp#L499

- https://github.com/ynput/OpenPype/blob/d958f989e05dd8c7d58733abacf0b1ab335cb763/openpype/hosts/tvpaint/api/pipeline.py#L69

Reply:
> Also:

That's a log, not visible by users.